include/sys/util.h

```diff
@@ -95,11 +95,13 @@ size_t ARRAY_SIZE(T(&)[N]) { return N; }
 #define ceiling_fraction(numerator, divider) \
 	(((numerator) + ((divider) - 1)) / (divider))
 
+#ifndef INLINE
 #ifdef INLINED
```

Comment:
was this originally a typo? (INLINED vs INLINE) This has been there forever, IMO we should just remove it and use inline directly in the one spot where this is being used.

Reply:
my guess it that the INLINED was not meant as a guard but a way of defining INLINE empty
I'm perfectly fine with removing it if you don't mind the "API breakage"